customTests/azure_ib_write_bw.nhc

```diff
@@ -51,7 +50,7 @@ do
 done
 if [[ $ib_bandwidth < $EXP_IB_BW ]]; then
       log "$IB_WRITE_BW_OUT2"
-      die 1 "$FUNCNAME: $IB_WRITE_BW, IB=mlx5_ib${device}, mlx5_ib${device_peer}, IB BW (expected > $EXP_IB_BW Gbps, but measured $ib_bandwidth Gbps"
+      die 1 "$FUNCNAME: $IB_WRITE_BW, IB=mlx5_ib0, mlx5_ib0, IB BW (expected > $EXP_IB_BW Gbps, but measured $ib_bandwidth Gbps"
```

Comment:
Should users be able to specify devices from config files? May be beyond the scope of this PR.

Reply:
Yea lets keep it like this for now